versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = "~> 1.1"
+  required_version = "> 1.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.12"
+      version = ">= 4.0"
```

Comment:
The provider version should probably be >= 4.9, to include backwards compatibility fixes.